docs/bugs.json

```diff
@@ -1,4 +1,12 @@
 [
+    {
+        "name": "ExpExponentCleanup",
+        "summary": "Using the ** operator with an exponent of type shorter than 256 bits can result in unexpected values.",
+        "description": "Higher order bits in the exponent are not properly cleaned before the EXP opcode is applied if the type of the exponent expression is smaller than 256 bits. In that case, the result might be larger than expected if the exponent is assumed to lie within the value range of the type. Literal numbers as exponents are unaffected as are variables of type uint256.",
```

Comment:
Should the description mentation that this does not happen, if the base type is wider than the exponent type?

Reply:
Yes, I will add it.